src/cmd/manage_nsfs.js

```diff
@@ -244,7 +245,7 @@ async function fetch_bucket_data(argv, from_file) {
             data.s3_policy = JSON.parse(argv.bucket_policy.toString());
         }
     }
-    if (action === ACTIONS.UPDATE) {
+    if (action === ACTIONS.UPDATE || action === ACTIONS.DELETE) {
```

Comment:
Why do we need to omit on delete?

Reply:
@liranmauda Code is trying to fetch the bucket data for the delete also, Then only we can get the storage path for temp folder deletion.